contributor-docs/testing.md

```diff
@@ -80,20 +80,20 @@ We are slowly moving away from using `Jest` snapshots as a way to test visual ch
 
 #### Updating `theme-preval` snapshots
 
-If you need to update the `theme-preval` snapshots, you will need to run the following commands:
-
-```sh
-npm run build
-npm run test -- -u
-```
-
-After running that command, make sure to update the cache-busting timestamp in [src/theme-preval.js](https://github.com/primer/react/blob/main/src/theme-preval.js)
+If you need to update the `theme-preval` snapshots, make sure to first update the cache-busting timestamp in [src/theme-preval.js](https://github.com/primer/react/blob/main/src/theme-preval.js)
 
 ```diff
 - // Cache bust: 2022-02-23 12:00:00 GMT (This file is cached by our deployment tooling, update this timestamp to rebuild this file)
 + // Cache bust: 2023-02-24 12:00:00 GMT (This file is cached by our deployment tooling, update this timestamp to rebuild this file)
 ```
 
+After you will need to run the following commands and commit changes to the `themePreval.test.ts.snap` file:
+
+```sh
+npm run build
+npm run test -- -u
+```
+
```

Comment:
I'm not sure about this, but I think updating the cache-busting timestamp needs to be done **before** running the commands? 🤔 Otherwise the `themePreval.test.ts.snap` file will keep using the old timestamp. 

/cc @broccolinisoup

Reply:
Oh that makes sense! I hadn't gone through the process myself, I was just documenting after a conversation but that makes sense to me. Thanks for updating the docs 🙏🏻